### PR TITLE
[NCP-3262] some text added for the nginx directives and variables via systemConfigs

### PR DESCRIPTION
**Pull request by junheeahn**
Please check this PR.
Thanks.

### CODE REVIEW

File: en_US.json [hunk 1]
```diff
@@ -1064,17 +1071,26 @@
...
 	"hint.directive.secure_link_secret": "set up a built-in access control method",
+	"hint.directive.set_attack_state_cpu": "sets threshold for specified $server_name and random number between mintime and maxtime as expiration for resetting of the variable attack_state_cpu state",
+	"hint.directive.set_attack_state_req": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_req state",
+	"hint.directive.set_attack_state_rspbytes": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable",
+	"hint.directive.set_attack_state_tlshs": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_tlshs state",
 	"hint.directive.slice": "request big object in small pieces from origin",
 	"hint.directive.slice_ignore_etag": "allow slices to have different ETag values",
+	"hint.directive.slice_verify_header": "specify extra header to verify slice consistency in additional to etag",
```
Comment (chiraags) [inline en_US.json:1090]: Should be "in addition to etag" and not "in additional to etag"

Comment (junheeahn) [reply en_US.json]: I modified it. Thanks.

File: en_US.json [hunk 2]
```diff
@@ -1064,17 +1071,26 @@
...
 	"hint.directive.sanitize_accept_encoding": "preprocess Accept-Encoding header",
 	"hint.directive.satisfy": "how to combine multiple auth results",
 	"hint.directive.secure_link": "set up a built-in access control method",
 	"hint.directive.secure_link_md5": "set up a built-in access control method",
 	"hint.directive.secure_link_secret": "set up a built-in access control method",
+	"hint.directive.set_attack_state_cpu": "sets threshold for specified $server_name and random number between mintime and maxtime as expiration for resetting of the variable attack_state_cpu state",
+	"hint.directive.set_attack_state_req": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_req state",
+	"hint.directive.set_attack_state_rspbytes": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable",
+	"hint.directive.set_attack_state_tlshs": "sets threshold for specified server and random number between a range mintime and maxtime as expiration for resetting of the variable attack_state_tlshs state",
```
Comment (chiraags) [inline en_US.json:1087]: Can you please share where did you get the descriptions from?
Is it mentioned somewhere?

Comment (junheeahn) [reply en_US.json:1087]: I referred to the Cache Operation Handbook (https://docs.google.com/document/d/1T4NVOiiv_OlYA6nzDcoTm7MpQMBz5E1nr-W78_7GNiQ/edit#). Should I officially receive the description separately? If I have to, I will request the reporter through the issue comment.

Comment (junheeahn) [reply en_US.json:1087]: I will request a description. So I'm going to close this PR.

Comment (junheeahn) [reply en_US.json:1087]: I asked Reporter Jianghai Yao for a description.
Thank you for pointing that out. @chiraags

**Pull request closed without merging**